src/nvim/macros.h

```diff
@@ -154,6 +154,6 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
-#define RGB(r, g, b) ((r << 16) | (g << 8) | b)
+#define NVIM_RGB(r, g, b) ((r << 16) | (g << 8) | b)
```

Comment:
In fact I thought about it and the way we are using the macro we simply can't transform it into a function: it's used to generate constant values in the `color_name_table`. The C compiler wouldn't accept a function there afaik. One would need some C++11 magic for that, which is even uglier.

As a general guideline, if something _can_ be transformed from a macro to a `static inline`, that's great. But as we see, that's not always possible.

So, please forget what I said :).